### PR TITLE
(Wrapper)Refactor quantization method parameters

### PR DESCRIPTION
**Pull request by KazunoriSumiya**
## Pull Request Description:

- Removed the TPC selection variable
- Changed the TPC tuning parameter to sdsdp_version
- Changed the Wrapper argument (argument order)
- Changed to get_target_platform_capabilities_sdsp()
- Changed the test code

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: model_compression_toolkit/wrapper/constants.py [hunk 1]
```diff
@@ -13,13 +13,10 @@
 # limitations under the License.
 # ==============================================================================
 
+
```
Comment (kawakami-masaki0) [inline model_compression_toolkit/wrapper/constants.py]: Please delete unnecessary line

File: model_compression_toolkit/wrapper/mct_wrapper.py [hunk 2]
```diff
@@ -581,19 +544,17 @@ def quantize_and_export(self, float_model: Any,
...
             ...     param_items=param_items
             ... )
 
         """
         try:
             # Step 1: Initialize and validate all input parameters
-            self._initialize_and_validate(
-                float_model, representative_dataset, method, framework,
-                use_internal_tpc, use_mixed_precision)
+            self._initialize_and_validate( float_model, representative_dataset,
```
Comment (ueno-hiroshi002) [inline model_compression_toolkit/wrapper/mct_wrapper.py]: Delete unnecessary space.

self._initialize_and_validate( float_model
↓
self._initialize_and_validate(float_model

File: tests_pytest/keras_tests/unit_tests/wrapper/test_mct_wrapper_keras_unit.py [hunk 3]
```diff
@@ -112,54 +113,28 @@ def test_modify_params_non_existing_keys(self) -> None:
...
-#           wrapper = MCTWrapper()
-#           wrapper.use_internal_tpc = False
-#           
-#           # Expect exception when EdgeMDT TPC is not available
-#           with pytest.raises(Exception) as exc_info:
-#               wrapper._get_tpc()
-#
-#           # Verify correct error message
-#           expected_msg = "EdgeMDT TPC module is not available."
-#           assert expected_msg in str(exc_info.value)
```
Comment (kawakami-masaki0) [inline tests_pytest/keras_tests/unit_tests/wrapper/test_mct_wrapper_keras_unit.py]: Please delete unnecessary test

File: tests_pytest/pytorch_tests/unit_tests/wrapper/test_mct_wrapper_pytorch_unit.py [hunk 4]
```diff
@@ -112,54 +111,28 @@ def test_modify_params_non_existing_keys(self) -> None:
         assert 'non_existing_key' not in wrapper.params
         assert 'another_fake_key' not in wrapper.params
 
-#    @patch('model_compression_toolkit.wrapper.mct_wrapper.mct.get_target_platform_capabilities')
```
Comment (kawakami-masaki0) [inline tests_pytest/pytorch_tests/unit_tests/wrapper/test_mct_wrapper_pytorch_unit.py]: Please delete unnecessary test

File: tests_pytest/pytorch_tests/unit_tests/wrapper/test_mct_wrapper_pytorch_unit.py [hunk 5]
```diff
@@ -112,54 +111,28 @@ def test_modify_params_non_existing_keys(self) -> None:
...
+        Test _get_tpc method.
+        
+        Verifies that the wrapper correctly calls
+        mct.get_target_platform_capabilities_sdsp with expected parameters.
+        
+        Note: Patch targets mct.get_target_platform_capabilities_sdsp because
+        MCTWrapper imports 'model_compression_toolkit as mct'.
+        """
+        wrapper = MCTWrapper()
+        wrapper.framework = 'tensorflow'
```
Comment (kawakami-masaki0) [inline tests_pytest/pytorch_tests/unit_tests/wrapper/test_mct_wrapper_pytorch_unit.py]: I think 'pytorch' is correct.

File: model_compression_toolkit/wrapper/mct_wrapper.py [hunk 6]
```diff
@@ -84,8 +80,7 @@ def __init__(self):
            :header: "Parameter Key", "Default Value", "Description"
            :widths: 30, 30, 40
```
Comment (kawakami-masaki0) [inline model_compression_toolkit/wrapper/mct_wrapper.py:81]: Do you include a "header" or "width" to create API documentation?

File: model_compression_toolkit/wrapper/constants.py (hunk 1 above)
Comment (KazunoriSumiya) [reply model_compression_toolkit/wrapper/constants.py]: delete unnecessary line

File: model_compression_toolkit/wrapper/mct_wrapper.py (hunk 2 above)
Comment (KazunoriSumiya) [reply model_compression_toolkit/wrapper/mct_wrapper.py]: Delete unnecessary space.

File: tests_pytest/pytorch_tests/unit_tests/wrapper/test_mct_wrapper_pytorch_unit.py (hunk 5 above)
Comment (KazunoriSumiya) [reply tests_pytest/pytorch_tests/unit_tests/wrapper/test_mct_wrapper_pytorch_unit.py]: correct to  'pytorch'

File: model_compression_toolkit/wrapper/mct_wrapper.py (hunk 6 above)
Comment (KazunoriSumiya) [reply model_compression_toolkit/wrapper/mct_wrapper.py:81]: Yes. This is the syntax to generate a table with sphinx.

**Review by ueno-hiroshi002 — APPROVED** [review]

File: tests_pytest/conftest.py [hunk 7]
```diff
@@ -13,9 +13,7 @@
 # limitations under the License.
 # ==============================================================================
 from unittest.mock import Mock
-
 from pytest import fixture
-
```
Comment (kawakami-masaki0) [inline tests_pytest/conftest.py:18]: Does this file need to be modified?

Comment (KazunoriSumiya) [reply tests_pytest/conftest.py:18]: Undo the change that removed the unnecessary lines
